src/utils/gaa.js

```diff
@@ -38,6 +38,8 @@ import '../model/doc';
 import {AnalyticsEvent, EventOriginator} from '../proto/api_messages';
 import {showcaseEventToAnalyticsEvents} from '../runtime/event-type-mapping';
 
+const jsonwebtoken = require('jsonwebtoken');
```

Comment:
jsonwebtoken is a dev dependency - in general we don't include non-google code in the production binary (see package.json - the only external deps are google authrored).  Can we avoid using it here?

Reply:
Nevermind - jasonwebtoken is an approved lib - you do need to move it to `dependencies`

Reply:
For future ref check g3/third_party to see if we already use a module and also use [jsgl](https://github.com/google/js-green-licenses) to confirm it's an ok license (we're good in this case)

Reply:
Hold on approving I'm checking on exactly how we need to include this - see email you are about to get.

Reply:
@ChrisAntaki mentioned that there is actually already a decoding function in our code, so I think there is no need to use this library.